Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository = "https://github.com/rustasync/tide"
 version = "0.2.0"
 
 [dependencies]
+bytes = "0.4.12"
```

Comment:
I wonder what is this used for?

Reply:
Ah, that can be removed safely. It was a remnant of some stuff I tried several days ago trying to write in the compression/decompression logic directly -- now all of it is in https://github.com/rustasync/async-compression-rs.

I'll go ahead and remove it real quick.

Reply:
This was added back because it turns out I needed it for testing decompression: https://github.com/rustasync/tide/blob/30a3b1404f6b27fdce8bcd97fc04124706b9f684/src/middleware/compression.rs#L253

Apparently, Travis and rustfmt don't like it when it's included in dev-dependencies, likely because it's already included through `http-service`, so it's in the regular dependencies. Not that it should matter much though, aside from it being exposed I suppose.